packages/wrangler/src/dev/miniflare.ts

```diff
@@ -109,7 +109,7 @@ export interface ConfigBundle {
 	localProtocol: "http" | "https";
 	localUpstream: string | undefined;
 	inspect: boolean;
-	serviceBindings: Record<string, (request: Request) => Promise<Response>>;
+	serviceBindings: Record<string, (_request: Request) => Promise<Response>>;
```

Comment:
No big deal in this PR, but when these function types have param names which are unused (sometimes they are used with `typeof`, but rarely) we should just use `_` as the name